src/cmd/run.go

```diff
@@ -33,9 +34,10 @@ import (
 
 var (
 	runFlags struct {
-		container string
-		distro    string
-		release   string
+		container    string
+		distro       string
+		release      string
+		preserve_fds int
```

Comment:
Nitpick: camel case is the norm in Go, so something like `preserveFDs` will be better.

Reply:
Done.

Reply:
Thanks.